src/main/java/frc/robot/extras/utils/GeomUtil.java

```diff
@@ -13,11 +8,30 @@
 import edu.wpi.first.math.geometry.Transform2d;
 import edu.wpi.first.math.geometry.Transform3d;
 import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.math.geometry.Twist2d;
+import edu.wpi.first.math.geometry.Twist3d;
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.Constants.FieldConstants;
 
 /** Geometry utilities for working with translations, rotations, transforms, and poses. */
 public class GeomUtil {
+  public static final Translation2d TRANSLATION2D_ZERO = new Translation2d();
+  public static final Translation2d TRANSLATION2D_CENTER =
+      new Translation2d(
+          FieldConstants.FIELD_LENGTH_METERS / 2.0, FieldConstants.FIELD_WIDTH_METERS / 2.0);
+  public static final Rotation2d ROTATION2D_ZERO = new Rotation2d();
+  public static final Rotation2d ROTATION2D_PI = Rotation2d.fromDegrees(180.0);
+  public static final Pose2d POSE2D_ZERO = new Pose2d();
+  public static final Pose2d POSE2D_CENTER = new Pose2d(TRANSLATION2D_CENTER, ROTATION2D_ZERO);
+  public static final Transform2d TRANSFORM2D_ZERO = new Transform2d();
+  public static final Translation3d TRANSLATION3D_ZERO = new Translation3d();
+  public static final Rotation2d ROTATION3D_ZERO = new Rotation2d();
+  public static final Transform3d TRANSFORM3D_ZERO = new Transform3d();
+  public static final Pose3d POSE3D_ZERO = new Pose3d();
+  public static final Twist2d TWIST2D_ZERO = new Twist2d();
```

Comment:
what is a twist

Reply:
It's used to measure a difference between two poses. https://first.wpi.edu/wpilib/allwpilib/docs/development/java/edu/wpi/first/math/geometry/Twist2d.html